app/src/main/java/org/schabi/newpipe/RouterActivity.java

```diff
@@ -114,6 +116,19 @@ public class RouterActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(final Bundle savedInstanceState) {
+        ThemeHelper.setDayNightMode(this);
+        setTheme(ThemeHelper.isLightThemeSelected(this)
+                ? R.style.RouterActivityThemeLight : R.style.RouterActivityThemeDark);
+        Localization.assureCorrectAppLanguage(this);
+
+        // Pass-through touch events to background activities
+        // so that our transparent window won't lock UI in the mean time
```

Comment:
Note that this does not work on Android 13 / API 33, but it does work e.g. on API 22, since recent Android versions made some changes regarding transparency. Maybe add a comment.